Branch-SDK/src/main/java/io/branch/referral/BillingGooglePlay.kt

```diff
@@ -0,0 +1,243 @@
+package io.branch.referral
+
+import android.content.Context
+import android.util.Log
+import com.android.billingclient.api.*
+import io.branch.indexing.BranchUniversalObject
+import io.branch.referral.util.*
+
+class BillingGooglePlay {
+
+    private lateinit var context_: Context
+    var billingClient: BillingClient? = null
+
+    fun createBillingClient(context: Context, callback: (Boolean) -> Unit) {
```

Comment:
split out the init from the `startConnection`

Think of this class as a thin wrapper over this client.